docs/guidelines.md

```diff
@@ -113,7 +113,30 @@ ngeo.exampleDirective = function(…) {
       // …
     },
     controllerAs: 'ctrl',
+    // …
+  });
+```
+
+But if you bind your directive to the controller and you declare your attribute
+as a two ways binding attrribute, you can use the `.` notation. See the example
```

Comment:
How does the compiler know that the properties defined in `scope` relate to the instance properties? I think it is by chance that it works for the example you have linked to.